server.go

```diff
@@ -105,11 +105,11 @@ type Athlete struct {
 
 // Activity represents the interesting fields from the list activty endpoint
 type Activity struct {
-	ID        int     `json:"ID"`
-	Distance  float32 `json:"distance"`
+	ID                 int     `json:"ID"`
```

Comment:
`gofmt` is your friend :)
Depending on your editor, you can set up `go fmt` to run on save. I use Sublime and have auto-imports and gofmt and auto-deletion of extra whitespace set up by default.